### PR TITLE
add mkldnn conv_transpose_bias fuse pass ut

### PR DESCRIPTION
**Pull request by baoachun**
### PR types
Others

### PR changes
Others

### Describe

新增mkldnn conv_transpose_bias fuse pass单测

### CODE REVIEW

**Comment by zhenlin-work** [comment]
> ### PR types
> Others
> 
> ### PR changes
> Others
> 
> ### Describe
> 新增mkldnn conv_transpose_bias fuse pass单测

File: python/paddle/fluid/tests/unittests/ir/inference/test_mkldnn_conv_transpose_bias_fuse_pass.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    def is_program_valid(self, program_config: ProgramConfig) -> bool:
+        attrs = [
+            program_config.ops[i].attrs
+            for i in range(len(program_config.ops))
+        ]
+
+        if attrs[0]['data_format'] == "NHWC":
+            return False
+        if attrs[1]['axis'] == 3:
+            return False
```
Comment (jiweibo) [inline python/paddle/fluid/tests/unittests/ir/inference/test_mkldnn_conv_transpose_bias_fuse_pass.py]: axis=3的时候模型不合法吗？

**Review by XieYunshen — APPROVED** [review]
LGTM for `set_tests_properties(test_mkldnn_conv_transpose_bias_fuse_pass PROPERTIES TIMEOUT 100)`

**Review by jiweibo — APPROVED** [review]
LGTM

**Pull request merged**